manifests/server.pp

```diff
@@ -404,11 +409,16 @@
           }
           '3.0': {
             exec { "renew crl.pem on ${name}":
-              command  => ". ./vars && EASYRSA_REQ_CN='' EASYRSA_REQ_OU='' openssl ca -gencrl -out ${server_directory}/${name}/crl.pem -config ${server_directory}/${name}/easy-rsa/openssl.cnf",
+              command  => "./easyrsa gen-crl && cp ./keys/crl.pem ${server_directory}/${server}/crl.pem",
```

Comment:
does this still work with easy-rsa 2 or does it already drop support for it?

Reply:
No, this is easy-rsa 3.0 exclusive which is fine at the line, because the exec resource is only executed if easyrsa_version is 3.0. Take note of the context it's inside a switch/case.